pages/api/slates/get.js

```diff
@@ -26,27 +26,14 @@ export default async (req, res) => {
     });
   }
 
-  if (Array.isArray(req.body.data.id)) {
-    const responseMultiple = await Data.getSlatesByIds({ ids: req.body.data.id });
-    if (!responseMultiple) {
-      return res.status(404).send({ decorator: "SERVER_GET_SLATES_NOT_FOUND", error: true });
-    }
-
-    if (responseMultiple.error) {
-      return res.status(500).send({ decorator: "SERVER_GET_SLATES_NOT_FOUND", error: true });
-    }
-
-    return res.status(200).send({ decorator: "SERVER_GET_SLATES", slate: responseMultiple });
-  } else {
-    const response = await Data.getSlateById({ id: req.body.data.id });
-    if (!response) {
-      return res.status(404).send({ decorator: "SERVER_GET_SLATE_NOT_FOUND", error: true });
-    }
-
-    if (response.error) {
-      return res.status(500).send({ decorator: "SERVER_GET_SLATE_NOT_FOUND", error: true });
-    }
+  const response = await Data.getSlateById({ id: req.body.data.id });
```

Comment:
This is confusing, this old endpoint seems to have nothing wrong with it?

Reply:
I don't think you need to modify this endpoint at all. You should just keep it the way it is.

Reply:
hmmm so original I set up this for explore. 
But I thought it might be better to separate it out bc it errors on line 7 when passing in an array.

Reply:
let me double check with Martina on this one

Reply:
Yeah don't change this. I think other things use the array version so this will break it. Basically I made this endpoint so that it works for both get (single) and get (multiple). Can you show me how it's erroring?